### DISCUSSION TITLE
是否支持自定义和调用高性能C++计算函数？

### DISCUSSION THREAD

**Discussion by Bingoang** [TuGraph-family/tugraph-db#174] [Q&A]
是否支持自定义和调用高性能C++计算函数？如果能的话，是只能在API层自定义和调用C++函数（这样的话速度会比较慢），还是可以在计算层、图存储层自定义和调用C++函数？

**Comment by knightast** [comment]
都是用存储过程，由于是在server端执行，所以性能没问题，SNB的测试就是这么跑的
